Gems/ImGui/Code/Source/ImGuiManager.h

```diff
@@ -104,6 +104,9 @@ namespace ImGui
         bool m_useLastPrimaryTouchPosition = false;
         bool m_simulateBackspaceKeyPressed = false;
 
+        bool m_activationBroadcasted = false;
+        bool m_deactivationBroadcasted = false;
```

Comment:
suggestion: Rather than two bools (that can get out of sync) this may be better represented as an enum with several states and only one single variable, this could make things a lot easier to maintain and understand in future.